main/getting-started/deploying.md

```diff
@@ -1,63 +1,91 @@
 # Deploying Smart Contracts
 
-The deploy command in the [agoric command line tool](https://agoric.com/documentation/getting-started/agoric-cli-guide.html#agoric-deploy) 
+The `agoric deploy` command in the [agoric command line tool](https://agoric.com/documentation/getting-started/agoric-cli-guide.html#agoric-deploy) 
 supports deploying contracts and off-chain web applications that talk to them. The command 
 has two primary uses:
 
 * Deploy smart contract source code onto the blockchain
-* Deploy and setup an application program to the local server
+* Deploy and setup an application program to a local server running an Agoric api process
 
-The deployment process uses the `agoric deploy` command to run your Dapp's `contract/deploy.js` and `api/deploy.js` scripts. You can use the deploy scripts created when you copied an existing Dapp into your directory as they are, or you can modify the scripts as suggested later in this document.
+Use the `agoric deploy` command to run your Dapp's `contract/deploy.js` 
+and `api/deploy.js` scripts. You can use the deploy scripts created when you copied an existing 
+Dapp into your directory as they are, or you can modify the scripts as suggested later in this document.
+
+Remember, your Dapp has three primary subdirectories:
+- `contract/`which contains files relating to your smart contract itself.
+- `api/`which contains **tyg todo: Could use suggestions for a concise description**.
```

Comment:
I *think* this is the wallet's API. When you want to add support for the wallet to talk to the contract or the UI, those changes would go here.

I'm expecting that @katelynsills  or @michaelfig  will correct me if that's wrong.

Reply:
The api/ folder has the dapp deploy scripts - basically everything you need to start the dapp contract instance and dapp backend. This has nothing to do with the wallet API.

Reply:
And yet another perspective:

- `api/` which is the "middle tier" that would run on a public Internet server to enable the UI frontend to communicate via HTTP/WebSocket to an instance of your on-chain backend contract.